Generators/include/Generators/PrimaryGenerator.h

```diff
@@ -0,0 +1,81 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \author R+Preghenella - June 2017
+
+#ifndef ALICEO2_EVENTGEN_PRIMARYGENERATOR_H_
+#define ALICEO2_EVENTGEN_PRIMARYGENERATOR_H_
+
+#include "FairPrimaryGenerator.h"
+
+class TFile;
+class TTree;
+class TString;
+class FairMCEventHeader;
+
+namespace o2
+{
+namespace eventgen
+{
+
+  /*****************************************************************/
+  /*****************************************************************/
+  
+  class PrimaryGenerator : public FairPrimaryGenerator
```

Comment:
what is a PrimaryGenerator? What is the difference to "Generator"?

Reply:
ok I see. This is mirroring the Fair(Primary)Generator semantics.

Reply:
this header file was added to this PR by mistake, will be removed